### PR TITLE
Updm rule

### PR DESCRIPTION
**Pull request by MikeMillerGIS**
fixed a few rules in telecom and got the Apply All pyt working again.

@jalsup I disable the ID equipment rule, was that working or tested?

### CODE REVIEW

File: Industry/UPDM/Calculation/Device-Cathodic_Protection_Traceability.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
+// Assigned To: PipelineDevice
+// Type: Calculation
+// Name: Cathodic Protection Traceability for Pipeline Devices
+// Description: Set traceability flag for device assets
+// Subtypes: All
+// Field: cptraceability
+// Trigger: Insert, Update
+
```
Comment (ted-howard) [inline Industry/UPDM/Calculation/Device-Cathodic_Protection_Traceability.js:8]: Add Exclude from Client and Disable setting. And to all rules?

File: Industry/UPDM/Calculation/Junction-Auto_Contain_By_Rules.js [hunk 2]
```diff
@@ -6,14 +6,44 @@
 // Field: assetid
 // Trigger: Insert, Update
 // Exclude From Client: True
+// Disable: False
+
+// Implementation Note: This rule is disabled by default as it auto performs containment of features when editing.  If you desire these functions, enable this rule and adjust the settings below.
```
Comment (ted-howard) [inline Industry/UPDM/Calculation/Junction-Auto_Contain_By_Rules.js:11]: Needs to be set to True to disable

File: Industry/UPDM/Calculation/Device-Auto_Contain_By_Rules.js [hunk 3]
```diff
@@ -6,14 +6,44 @@
 // Field: assetid
 // Trigger: Insert, Update
 // Exclude From Client: True
+// Disable: False
+
+// Implementation Note: This rule is disabled by default as it auto performs containment of features when editing.  If you desire these functions, enable this rule and adjust the settings below.
```
Comment (ted-howard) [inline Industry/UPDM/Calculation/Device-Auto_Contain_By_Rules.js:11]: Needs to be set to True to disable

File: Industry/UPDM/Calculation/Junction-Auto_Contain_By_Rules.js [hunk 4]
```diff
@@ -6,14 +6,44 @@
...
+
+// Related Rules: Some rules are rely on additional rules for execution.  If this rule works in conjunction with another, they are listed below:
```
Comment (ted-howard) [inline Industry/UPDM/Calculation/Junction-Auto_Contain_By_Rules.js:13]: "Some rules rely" - this is on every rule

File: Industry/UPDM/Calculation/Junction-Auto_Contain_By_Rules.js [hunk 5]
```diff
@@ -6,14 +6,44 @@
...
 var assigned_to_field = $feature.assetid;
+
+// The association status field, this field is maintained by the utility network
+// ** Implementation Note: Do not change this value, it is in this section in case there is a schema change to the utility network in the future.
 var association_status = $feature.associationstatus;
-var search_distance = 40; //DefaultValue($feature.searchdistance, 75);
+
+// The distance to search for a container
+// ** Implementation Note: Adjust this value based on the distance to auto contain, the comment after the value is an example how to use a
+//    field to defined the distance, this field does not exist in the data model, but could be added to provide a data specific distance.
```
Comment (ted-howard) [inline Industry/UPDM/Calculation/Junction-Auto_Contain_By_Rules.js:34]: change to "field to define the distance"

File: Industry/UPDM/Calculation/Junction-Auto_Contain_By_Rules.js [hunk 6]
```diff
@@ -6,14 +6,44 @@
...
+// The unit of the distance value in search_distance.
+// ** Implementation Note: Set the distance to search, for unit codes - https://desktop.arcgis.com/en/arcobjects/latest/net/webframe.htm#esriSRUnitType.htm
 var search_unit = 9002;
 
+// The utility network class ID for each layer.
+// ** Implementation Note: Adjust these values to match your geodatabase.  The values defined below are for a Version 3
+//    utility network with only the schema provide(no additional domain networks)
+//    These values will change with a version 4 utility network or if you applied more than one domain to the same utility network.
+//    To review these values, open the attribute domains in the database with the utility network and search for FeatureSourceID.  This domain
+//    will list the class id by un domain alias, not class name, so you will have to related the un attribute domain to the gdb class name to match its ID
```
Comment (ted-howard) [inline Industry/UPDM/Calculation/Junction-Auto_Contain_By_Rules.js:46]: "relate" not related

File: Industry/UPDM/Calculation/Junction-Auto_Contain_By_Rules.js [hunk 7]
```diff
@@ -58,6 +91,9 @@ function get_features_switch_yard(class_name, fields, include_geometry) {
     }
     return feature_set;
 }
+
+// A function to determine if the edit is triggered by UpdateSubnetwork.  IF a filegeodatbase, this action triggers an edit event, it enterprise, triggering the event is optional
```
Comment (ted-howard) [inline Industry/UPDM/Calculation/Junction-Auto_Contain_By_Rules.js:95]: If a file geodatabase

File: Industry/UPDM/Calculation/Junction-Auto_Contain_By_Rules.js [hunk 8]
```diff
@@ -58,6 +91,9 @@ function get_features_switch_yard(class_name, fields, include_geometry) {
...
+// ** Implementation Note: If using the industry model with all tiers, not change is required.  If a teir was removed, remove the match if block.
```
Comment (ted-howard) [inline Industry/UPDM/Calculation/Junction-Auto_Contain_By_Rules.js:96]: tier not teir

**Comment by MikeMillerGIS** [comment]
Thanks @ted-howard  all updated

**Pull request merged**